lib/lib-table-cluster/src/main/java/org/sagebionetworks/table/cluster/SQLUtils.java

```diff
@@ -2046,4 +2047,63 @@ public static String getViewScopeSubTypeFilter(ViewScopeFilter scopeFilter) {
 		
 		return builder.toString();
 	}
+
+	public static String appendAdditionalFilters(List<QueryFilter> additionalFilters){

```

Comment:
Consider move this to the SQLTranslatorUtils and rename it to translateQueryFilters